sdk/python/feast/client.py

```diff
@@ -668,9 +673,13 @@ def get_online_features(
                 "feature_set:feature" where "feature_set" & "feature" refer to
                 the feature and feature set names respectively.
                 Only the feature name is required.
-            entity_rows: List of GetFeaturesRequest.EntityRow where each row
+            entity_rows:
+                List of GetFeaturesRequest.EntityRow where each row
                 contains entities. Timestamp should not be set for online
                 retrieval. All entity types within a feature
+                OR
+                List of Dict[str, Union[bool,bytes,float,int,str,List[bool,bytes,float,int,str]]]]
```

Comment:
Can also allow users to pass `Dict[str,feast.types.Value]` when they want to specify type information post v0.7, as we don't have a native Python equivalent of corner types like `int32` or `float`.

Reply:
So explicit declaration of something like `Value(int64_val=1001)`? Should that be included in this PR?

Reply:
Yes so something like:
```python
client.get_online_features(
    ...,
    entity_rows={
        "customer_id": Value(int32_val=1001),
    }
)
```
so we support `Value` as well as native types, as there is no native equivalent for some types ie `float` and `int32`.

Reply:
I think this is fair.

Reply:
@terryyylim can you please add a test for thes Value() types?